tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -77,6 +82,48 @@ def environment_length_bytes(e: dict) -> int:
     return string_length_bytes(serialized_environment)
 
 
+class TestRuntimeValidation:
+    @markers.aws.only_localstack
+    def test_create_deprecated_function_runtime_with_validation_disabled(
+        self, create_lambda_function, lambda_su_role, aws_client, monkeypatch
+    ):
+        monkeypatch.setattr(config, "LAMBDA_RUNTIME_VALIDATION", 0)
+        function_name = f"fn-{short_uid()}"
+        create_lambda_function(
+            handler_file=TEST_LAMBDA_PYTHON_ECHO,
+            func_name=function_name,
+            runtime=Runtime.python3_7,
+            role=lambda_su_role,
+            MemorySize=256,
+            Timeout=5,
+            LoggingConfig={
+                "LogFormat": LogFormat.JSON,
+            },
+        )
+
+    @markers.aws.validated
+    @pytest.mark.parametrize("runtime", DEPRECATED_RUNTIMES)
+    def test_create_deprecated_function_runtime_with_validation_enabled(
+        self, runtime, create_lambda_function, lambda_su_role, aws_client, monkeypatch, snapshot
+    ):
+        monkeypatch.setattr(config, "LAMBDA_RUNTIME_VALIDATION", 1)
+        function_name = f"fn-{short_uid()}"
+
+        with pytest.raises(aws_client.lambda_.exceptions.InvalidParameterValueException) as e:
+            create_lambda_function(
```

Comment:
Rather than using fixture, I should switch to caling lambda client directly, in order to avoid retry and speed up tests

Reply:
That would be nice indeed!

Reply:
Sounds good. I'll add it :)

Reply:
Done